config/locales/models/motif.fr.yml

```
@@ -82,13 +85,30 @@ fr:
       motif/location_types/hint:
         public_office: Le RDV aura lieu au lieu sélectionné (MDS...)
```

et ici aussi on peut améliorer le texte 
```suggestion
        public_office: L'usager se rendra sur place, au lieu sélectionné (MDS...)
```

---

Décision du trio : on préfère s'aligner sur l'existant.